src/Testcontainers/Configurations/Modules/Databases/MySqlTestcontainerConfiguration.cs

```diff
@@ -11,6 +11,9 @@ public class MySqlTestcontainerConfiguration : TestcontainerDatabaseConfiguratio
 
     private const int MySqlPort = 3306;
 
+    private const string RootUsername = "root";
+    private string _username = "root";
```

Comment:
Can we please remove these unnecessary fields? Usually we store the values directly in the dictionary.

Reply:
see [3aa809e](https://github.com/testcontainers/testcontainers-dotnet/pull/629/commits/3aa809ead984eaac32b08c927bb0bbb37d4741c8)